app/models/miq_ae_method.rb

```diff
@@ -33,6 +33,10 @@ def self.available_scopes
     AVAILABLE_SCOPES
   end
 
+  def self.available_expression_objects
+    Rbac::Filterer::CLASSES_THAT_PARTICIPATE_IN_RBAC
```

Comment:
@mkanoor This list is not the complete set I would expect to see.  For example it does not show `Vm`, just the base model `VmOrTemplate`.  I think it would be better to expose the list from the MiqExpression model `MiqExpression.base_tables `

That is the list Reporting uses: https://github.com/ManageIq/manageiq/blob/5cf8d1ffd0e9489f9a58ecb1223c3a816817dd76/app/models/miq_report.rb#L107